templates/triton_services/README.md

```diff
@@ -0,0 +1,201 @@
+onto the model repository directory. Then compile the model into TensorRT engine
+and also store the artifacts in the model repository directory. For more details,
+you can check the `stable_diffusion_pipeline.py` file in the `diffusion` directory.
+"""
+model = triton_server.load("stable_diffusion_1_5")
+duration = time.time() - t0
+print(f"Total duration: {duration}s")
+
+# Unload the model and the server to free the memory.
+triton_server.unload(model, wait_until_unloaded=True)
+triton_server.stop()
+```
+
+After the model compile, you can upload the model to a cloud storage. There are some
+.py and onnx files generated by the previous step in the same model repository.
+You only need to upload both the model config file `config.pbtxt` and the TensorRT
+engine artifacts from the model repository. Anyscale provides an environment variable
+`ANYSCALE_ARTIFACT_STORAGE` for customers to store model artifacts. If you want to
+learn more about the storage, you can check the 
+[doc](https://docs.anyscale.com/1.0.0/services/storage/#object-storage-s3-or-gcs-buckets).
+Use one of the following to upload the model.
+
+
+<Tabs>
+    <TabItem value="AWS" label="AWS" default>
+
+```python
+aws s3 cp /tmp/workspace/diffusion-models/stable_diffusion_1_5/config.pbtxt $ANYSCALE_ARTIFACT_STORAGE/triton_model_repository/stable_diffusion_1_5/config.pbtxt
+aws s3 cp /tmp/workspace/diffusion-models/stable_diffusion_1_5/1/1.5-engine-batch-size-1/ $ANYSCALE_ARTIFACT_STORAGE/triton_model_repository/stable_diffusion_1_5/1/1.5-engine-batch-size-1/ --recursive
+```
+
+  </TabItem>
+  <TabItem value="GCP" label="GCP">
+
+```python
+gcloud storage cp /tmp/workspace/diffusion-models/stable_diffusion_1_5/config.pbtxt $ANYSCALE_ARTIFACT_STORAGE/triton_model_repository/stable_diffusion_1_5/config.pbtxt
+gcloud storage cp /tmp/workspace/diffusion-models/stable_diffusion_1_5/1/1.5-engine-batch-size-1/ $ANYSCALE_ARTIFACT_STORAGE/triton_model_repository/stable_diffusion_1_5/1/1.5-engine-batch-size-1/ --recursive
+```
+
+  </TabItem>
+</Tabs>
+
+## Run Triton Server on Ray Serve locally on Anyscale Workspaces
+
+The `triton_app.py` included in this workspace demonstrates using
+remote model repository to start triton serve, loading the specific Stable Diffusion
+model, run inference with Triton, and serving the response through Ray Serve.
+You can view the file by clicking on the `triton_app.py` in the file explorer. In
+addition, you can also do prompt engineering, apply business logic, or doing model
+composition with Ray Serve before returning the response as image. Run the follow code
+to start Triton Server with Ray Serve.
+
+
+```python
+serve run triton_app:triton_deployment --non-blocking
+```
+
+Cluster autoscaler starts a GPU worker node. The model downloads from the
+cloud storage location where you just uploaded the model artifacts, and then loaded into
+the Triton Server and serve the endpoint with Ray Serve. It might take few minutes to
```

Comment:
```suggestion
the Triton Server and serve sthe endpoint with Ray Serve. It might take few minutes to
```

Reply:
assuming this is `the Triton Server and serve "an" endpoint with Ray Serve. It might take few minutes to`